src/Authorization.AspNetCore/Authorization.AspNetCore.csproj

```diff
@@ -10,6 +10,13 @@
 
   <ItemGroup Label="Package References">
     <PackageReference Include="GraphQL" Version="$(GraphQLVersion)" />
+  </ItemGroup>
+
+  <ItemGroup Label="Package references for .NET Core" Condition="'$(IsNetCore30OnwardsTarget)' == 'True'">
+    <FrameworkReference Include="Microsoft.AspNetCore.App" />
```

Comment:
It looks strange to refer from a library to a meta-package. It seems to me that the documentation talked about using this method for end applications, not libraries.

Reply:
See https://github.com/aspnet/AspNetCore/issues/6040 and https://github.com/aspnet/AspNetCore/issues/3756. It's just the way it is now for 3.0. The dependencies that were previously exposed as separate packages are only available in the shared framework now so you don't have a choice.

Reply:
Wow, I did not notice that here `FrameworkReference ` instead of `PackageReference `.
Well, the dependencies issue is the first thing that bothered me. Now I understand that my concerns about shifting from fine grained netstandard2.0-targeted packages to one shared netcoreapp3-targeted framework were well founded. Now I understand what you meant when you said that netstandard is dead. Let's just say it is dead only for applications and libraries that are built on top of ASP.NET Core stack. Now I have gotten more work to migrate our applications to ASP.NET Core 3.0 - we will have to use **Trimmed** Self-Contained Deployment. Thanks for the useful information!

Reply:
Yea trimming will be the way to go if doing self-contained, that's for sure. 

I think it's probably going to be dead for non-aspnetcore stacks too, as .NET 5 will be .NET Core and only supported by `netcoreapp5.0` I'd imagine. I'm talking dead in a long way off though. As long as we support netfx it'll be useful, but come .NET 6 and 7, etc, I imagine the idea really is that netcoreapp is xplat enough that it can run everywhere. Whether that's the case though is yet to be seen. Potentially IoT and some targets will mean that netstandard is still useful over there.

Reply:
Let's not be upset ahead of time.